services/export/shared/csvExport.js

```diff
@@ -81,26 +101,38 @@ const validateIfAccountHasTransactions = async (params) => {
 	return !!response.data.length;
 };
 
-const parseTransactionsToCsv = (json) => {
-	const opts = { delimiter: config.csv.delimiter, fields };
-	return parseJsonToCsv(opts, json);
+const getDefaultStartDate = async () => {
+	const { data: [block] } = await getFirstBlock();
+	return moment(block.timestamp * 1000).format(DATE_FORMAT);
 };
 
+const getToday = () => moment().format(DATE_FORMAT);
+
 const standardizeIntervalFromParams = async ({ interval }) => {
 	let from;
 	let to;
 	if (interval && interval.includes(':')) {
 		[from, to] = interval.split(':');
+		if ((moment(to, DATE_FORMAT).diff(moment(from, DATE_FORMAT))) < 0) {
+			throw new ValidationException(`Invalid interval: ${interval}`);
+		}
 	} else if (interval) {
 		from = interval;
-		to = moment().format(config.csv.dateFormat);
+		to = getToday();
 	} else {
-		from = '2016-01-01'; // TODO: Start date of the blockchain
-		to = moment().format(config.csv.dateFormat);
+		from = await getDefaultStartDate();
+		to = getToday();
 	}
 	return `${from}:${to}`;
 };
 
+const getPartialFilenameFromParams = async (params) => {
+	const address = getAddressFromParams(params);
+	const [from, to] = (await standardizeIntervalFromParams(params)).split(':');
+	const filename = `${address}_${from}_${to}.json`;
+	return filename;
+};
```

Comment:
Let us modify the method as below, since partials are supposed to store the data only for a single day:
```suggestion
const getPartialFilenameFromParams = async (params, day) => {
	const address = getAddressFromParams(params);
	const filename = `${address}_${day}.json`;
	return filename;
};
```